django_mongodb/query_utils.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+
+def is_direct_value(node):
+    return not hasattr(node, "as_sql")
+
+
+def process_lhs(node, compiler, connection):
+    if is_direct_value(node.lhs):
+        return node
+    return node.lhs.as_mql(compiler, connection)
+
+
+def process_rhs(node, compiler, connection):
+    _, value = node.process_rhs(compiler, connection)
+    lookup_name = node.lookup_name
+    # Undo Lookup.get_db_prep_lookup() putting params in a list.
+    if lookup_name not in ("in", "range"):
+        value = value[0]
```

Comment:
All values come through as List[Any] types?

Reply:
See comment two lines above. This is copied from the old `_normalize_lookup_value()`.

Reply:
Well, yeah, the thing is:
We use a Django method to normalize things, those methods returns: (sql_string, [parameters]) And we only need the parameters, and there is an especial case with in and range, that [parameters] must keeps as an array. 
Django return the value in the way that sql_string % params works. So we need to unfold this structure.